### PR TITLE
Jberry updating latest models

### PR DESCRIPTION
**Pull request by jbsdoki**
Added standard image transformations to my 8 class and 5 class models and now acheiving much higher accuracies with different hyperparameters, validation accuracy around 45 - 65%, highest model 8 class acheived 75%

### CODE REVIEW

**Comment by jbsdoki** [comment]
Added standard image transformations to my 8 class and 5 class models and now acheiving much higher accuracies with different hyperparameters, validation accuracy around 45 - 65%, highest model 8 class acheived 75%
